lib_ecstasy/src/main/x/ecstasy/mgmt/BasicResourceProvider.x

```diff
@@ -1,3 +1,5 @@
+import ecstasy.annotations.Inject.Options;
```

Comment:
You added this import to use on line 63, but there other places that fully qualify it (lines 46 and 53). We should either change them all or qualify them all